theano/gpuarray/dnn.py

```diff
@@ -1213,8 +1215,9 @@ def dnn_conv3d(img, kerns, border_mode='valid', subsample=(1, 1, 1), dilation=(1
         out = GpuAllocEmpty(dtype=img.dtype, context_name=ctx_name)(*out_shp)
         precision, _ = get_precision(precision, [img, kerns], for_grad=True)
         desc = GpuDnnConvDesc(border_mode='valid', subsample=(1, 1, 1), dilation=(1, 1, 1),
+                              num_groups=num_groups,
```

Comment:
This is useless, line 1200 assert that num_groups is 1

Reply:
Yes, but for future inspection and making sure that num_groups is passed correctly it's much easier to always pass it than to depend on a condition some place above that makes sure it is 1.